flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/ExprCodeGenerator.scala

```diff
@@ -715,9 +715,9 @@ class ExprCodeGenerator(ctx: CodeGeneratorContext, nullableInput: Boolean)
       case ARRAY_VALUE_CONSTRUCTOR =>
         generateArray(ctx, resultType, operands)
 
-      // maps
-      case MAP_VALUE_CONSTRUCTOR =>
-        generateMap(ctx, resultType, operands)
+      // maps and multisets
+      case MAP_VALUE_CONSTRUCTOR | MULTISET_VALUE =>
```

Comment:
Let's also call it `MULTISET_VALUE_CONSTRUCTOR` to be in sync with map and array